flink-core/src/test/java/org/apache/flink/core/fs/FileSystemTest.java

```diff
@@ -23,20 +23,16 @@
 import org.apache.flink.core.fs.local.LocalFileSystem;
 import org.apache.flink.util.WrappingProxy;
 import org.apache.flink.util.WrappingProxyUtil;
-import org.apache.flink.util.function.ThrowingRunnable;
 
-import org.hamcrest.Matchers;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
 import java.io.IOException;
 import java.net.URI;
 import java.net.URISyntaxException;
 
-import static org.hamcrest.Matchers.containsString;
-import static org.hamcrest.Matchers.not;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 /** Tests for the {@link FileSystem} base class. */
 public class FileSystemTest {
```

Comment:
It seems that you have migrated your tests to junit5 by the way, which is really good.
I have two small suggestions:
1.juni5 it is not recommended to modify the test class and methods with pubic access modifier, but use the default modifier directly.
2. the assertion part should use Assertj instead of Junit's own.

Reply:
1. Changed all places
2. Not sure why Assertj instead of Junit. Is there an agreement in the dev channel about that?

Reply:
> 1. Changed all places
> 2. Not sure why Assertj instead of Junit. Is there an agreement in the dev channel about that?
You can reference to 
https://issues.apache.org/jira/browse/FLINK-25325
or flink code style
https://flink.apache.org/contributing/code-style-and-quality-common.html#testing

Reply:
Now only Assertj should be in place here.